readconfig.py

```diff
@@ -27,6 +27,15 @@
 from collections import OrderedDict
 from typing import Dict
 
+class bcolors:
```

Comment:
ANSI Escape Sequences are not enabled by default, and not supported before Windows 10 TH2 (v1511).
![image](https://user-images.githubusercontent.com/16814706/163305748-68312fd7-d3aa-4c96-ba0a-413ec03fc601.png)

Reply:
EE2 and EE3 examples

![image](https://user-images.githubusercontent.com/16814706/163307186-207f02e6-1764-4d9f-804c-64476e853256.png)
![image](https://user-images.githubusercontent.com/16814706/163307500-9c2477e3-d13d-4952-bab7-fa1fa3d32dee.png)